crates/neon/src/sys/bindings/functions.rs

```diff
@@ -209,7 +210,7 @@ mod napi1 {
 
             fn strict_equals(env: Env, lhs: Value, rhs: Value, result: *mut bool) -> Status;
 
-            #[cfg(feature = "external-buffers")]
+            #[cfg(any(feature = "sys", feature = "external-buffers"))]
```

Comment:
Why is this `any` and not `all`?

Reply:
The symbol will always exist in versions of Node-API that support it. Instead, it fails at runtime.

Since this is the `sys` crate, I wanted to make all symbols available, regardless of whether Neon uses them and hence the `any`.

Do you think we should require users to *also* enable the `external-buffers` feature for access to this `sys` API?

Reply:
Resolving this since we determined this to be correct. We don't want users to opt-in a 2nd time since this API is already unsafe.